servers/physics_3d/soft_body_3d_sw.cpp

```diff
@@ -243,7 +243,7 @@ void SoftBody3DSW::update_area() {
 		const Vector3 a = x1 - x0;
 		const Vector3 b = x2 - x0;
 		const Vector3 cr = vec3_cross(a, b);
-		face.ra = cr.length();
+		face.ra = cr.length() / 2.;
```

Comment:
Please use `* 0.5` instead of `/ 2.`. It's micro-optimization, but it might be worth it since this code is performance critical.

Reply:
Actually, the compiler is smart enough to optimise this either way. What's more important is that the code is written in a manner that makes the intention clear.

Reply:
Just seeing this @pouleyKetchoupp. No problem. I'll do that now.

Would it be best to collect any other issues that I find going through the Bullet soft body port into a single PR?

Reply:
@madmiraal That is not always true, according to this discussion:
https://stackoverflow.com/questions/35506226/will-the-compiler-optimize-division-into-multiplication
It might depend on the compiler and flags, so better do a change that has no cost and be safe, rather than spend extra time doing more research to figure out if it's true for all platforms and compilers supported by Godot.

The intention should be perfectly clear for any programmer when seeing `* 0.5` since it's trivial and widely used.

@jeffrey-cochran Yeah, you can definitely wait to see if there's similar things to add to this PR. Just make sure you squash your commits in the end (there's currently two).

Reply:
Got it. I'll ping when I feel I've done my due diligence for now